vscode-web/run.sh

```diff
@@ -2,6 +2,40 @@
 
 BOLD='\033[0;1m'
 EXTENSIONS=("${EXTENSIONS}")
+VSCODE_WEB="${INSTALL_PREFIX}/bin/code-server"
+
+# Set extension directory
+EXTENSION_ARG=""
+if [ -n "${EXTENSIONS_DIR}" ]; then
+  EXTENSION_ARG="--extensions-dir=${EXTENSIONS_DIR}"
+fi
+
+run_vscode_web() {
+  echo "👷 Running $VSCODE_WEB serve-local --port ${PORT} --host 127.0.0.1 --accept-server-license-terms serve-local --without-connection-token --telemetry-level ${TELEMETRY_LEVEL} in the background..."
```

Comment:
This is missing `$EXTENSIONS_ARG`.  In the future it might make sense to pass these args to a function that logs them and then calls the binary with them, so new arguments are not missed.  We also have `serve-local` twice.

```suggestion
  echo "👷 Running $VSCODE_WEB serve-local $EXTENSIONS_ARG --port ${PORT} --host 127.0.0.1 --accept-server-license-terms --without-connection-token --telemetry-level ${TELEMETRY_LEVEL} in the background..."
```